test/functional/apps/dashboard_elements/controls/options_list.ts

```diff
@@ -429,7 +496,7 @@ export default function ({ getService, getPageObjects }: FtrProviderContext) {
         });
 
         it('Does not mark selections invalid with Query', async () => {
-          await queryBar.setQuery('isDog : false ');
+          await queryBar.setQuery('NOT animal.keyword : "dog" ');
```

Comment:
Because `isDog` is a scripted field, it was causing flakiness with the new document that I added for the exists query tests. Using this comparison instead fixed the flakiness, and should be equivalent! :+1: